sunscreen/benches/fractional_range_proof.rs

```diff
@@ -140,5 +158,217 @@ fn fractional_range_proof(_c: &mut Criterion) {
     println!("Verifier time {}s", verifier_time.elapsed().as_secs_f64());
 }
 
-criterion_group!(benches, fractional_range_proof);
+/// Imagine a multi-party computation where each user submits a value between
+/// zero and a maximum encrypted under their key. The MPC computes the mean and
+/// variance of their inputs.
+/// 
+/// This proof is similar to [`private_tx_fractional_range_proof`] except we're only
+/// proving 0 < a <= b where
+/// 
+/// * a is the submitted value under a given user's key.
+/// * b is the maximum value encrypted under the same user's key.
+fn mean_variance_fractional_range_proof(_c: &mut Criterion) {
+    #[zkp_program(backend = "bulletproofs")]
+    /**
+     * Proves the 0 < a <= b and a == c
+     */
+    fn in_range<F: BackendField>(
+        a: [[NativeField<F>; 8]; 64],
+        b: [[NativeField<F>; 8]; 64],
+    ) {
+        println!("Running mean_variance_fractional_range_proof...");
+
+        let get_coeffs = |x: &[[ProgramNode<NativeField<F>>; 8]]| -> Vec<_> {
+            x.iter().map(|x| to_field_element(x, true)).collect()
+        };
```

Comment:
Since this is used more than once now (in `private_tx_fractional_range_proof`), might as well move it up to file level scope.